src/Output/HumanReadableDisplay.php

```diff
@@ -37,13 +37,13 @@ public function onOutcomeSuccess(): void
         $this->style->success('All dependencies have allowed licenses.');
     }
 
-    public function onPackageWithViolatingLicense(Package $package, string $license): void
+    public function onPackageWithViolatingLicense(Package $package): void
     {
         $this->style->error(
             \sprintf(
-                'Dependency "%s" has license "%s" which is not in the allowed list.',
+                'Dependency "%s" is licensed under "%s" which is not in the allowed list.',
```

Comment:
it will improve the UI for sure, will do that now, thank you

Reply:
done in 4d850a8b0799ecdfd41b372fd2f1c27cfc7d5ae6, I hope it looks good for you

I had to phrase it like `none of which are allowed` instead of `none of which are in the allowed list` because the longer version breaks into two lines in the terminal, because of the universal Symfony's limit of 120 characters per line.